MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -21,16 +22,13 @@ - (void) lookForMatches{
     PFObject *music = [PFObject objectWithClassName:@"Music"];
     PFQuery *query = [PFQuery queryWithClassName:@"Music"];
     PFUser *current = [PFUser currentUser];
+    self.author = [SpotifyAPIManager shared].author;
```

Comment:
Passes author and calls upon it in SpotifyApiManager to get the current user so I dont have to mkae a call back to get the information